package.nls.fr.json

```diff
@@ -0,0 +1,25 @@
+{
+  "command.regenerateIcons.title": "Appliquer Icônes Personnalisation",
```

Comment:
I would say that this translation is not good. It sounds like a direct English translation.I would say something like `Appliquer la personnalisation des icônes`